docs/src/pages/guides/notifications.md

```diff
@@ -80,19 +80,14 @@ Type of Notification in this case it would be set as `mailgun`.
 The detail settings needed to send a mailgun.
 
 **Recipients**
-Email addresses that will accept the mail sent from your mailgun account, the recipients address is set in array making it able to send to multiple addresses. `["monika@testmail.com", "symon@testmail.com"]`
+An array of email addresses that will receive the e-mail from Monika. ex. `["monika@testmail.com", "symon@testmail.com"]`
```

Comment:
List of recipient email addresses as a string array. e.g. `["monika@testmail.com", "symon@testmail.com"]`
<i>better use e.g. than ex. e.g. "exempli gratia" means "for example" in latin</i>

Reply:
updated, please check